consensus/state.go

```diff
@@ -32,6 +33,7 @@ type TimeoutParams struct {
 	Precommit0     int
 	PrecommitDelta int
 	Commit0        int
+	SkipCommit0    bool
```

Comment:
to be consistent with the naming above? to be honest, I like `SkipTimeoutCommit` more